frontend/packages/console-shared/src/components/synced-editor/index.tsx

```diff
@@ -37,6 +38,8 @@ export const SyncedEditor: React.FC<SyncedEditorProps> = ({
   const [formData, setFormData] = React.useState<K8sResourceKind>(initialData);
   const [yaml, setYAML] = React.useState(safeJSToYAML(initialData));
   const [type, setType] = React.useState<EditorType>(initialType);
+  const [safeToSwitch, setSafeToSwitch] = React.useState(true);
+  const [yamlWarning, setYAMLWarning] = React.useState(false);
 
```

Comment:
I believe our best practice is to specify:
```
React.useState<boolean>(...)
```

Reply:
@dtaylor113 It infers the type if you give it a default value. But it's definitely needed if you do something like `React.useState(null)` or `React.useState([])` where it can't infer the type.

Reply:
I thought this might be the case, but I added it anyway 😆